packages/static-wado-util/lib/image/bilinear.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Performs a bilinear scaling, both scaling up and scaling down.
+ * @param src - src image frame to get map from
+ * @param dest - dest image frame to write to
+ * @returns destination data buffer
+ */
+function bilinear(src, dest) {
+  const { rows: srcRows, columns: srcColumns, pixelData: srcData } = src;
```

Comment:
ideally these should be imported from dicom image loader later we merge right?

Reply:
Yes, there is an entire change to mvoe to CS3D rendering for thumbnails, at which point the library will be available.